src/components/helpers/utils.ts

```diff
@@ -41,9 +56,13 @@ export const getServiceShutteredStatus = async (): Promise<boolean> => {
     const urlConfig = new URL(
       `${sdkConfig.serverConfig.infinityRestServerUrl}/app/${sdkConfig.serverConfig.appAlias}/api/application/v2/data_views/D_ShutterLookup`
     ).href;
-    const featureID = isUnAuthJourney() ? 'UnauthChB' : 'ChB';
-    const featureType = 'Service';
-
+    
+    let featureID = 'ChB'; const featureType = 'Service';
```

Comment:
is prettier/lint working for you? It should be on separate line.

Reply:
Yes the linting is working but not pointing this one as its formatting one.
done it through prettier now.